Source/Core/Core/HW/DSPLLE/DSPSymbols.h

```diff
@@ -9,7 +9,11 @@
 #include "Common/CommonTypes.h"
 #include "Common/SymbolDB.h"
 
-namespace DSPSymbols
+namespace DSP
+{
+namespace LLE
+{
+namespace Symbols
```

Comment:
This is probably similar to the Profile, but does this have to be inside the LLE namespace? I'd say Symbols and the Profiler aren't really LLE specific (albeit only being used in LLE at this point) and could be moved elsewhere.